pages/common/az-login.md

```diff
@@ -0,0 +1,29 @@
+# az login
+
+> Login to Azure.
+> Part of `az`.
+> More information: <https://docs.microsoft.com/cli/azure/authenticate-azure-cli>.
```

Comment:
Why did you choose this link? If you choose the reference page for `az-logout` I think you should use this here:
```suggestion
> More information: <https://docs.microsoft.com/cli/azure/reference-index#az_login>.
```

Reply:
Ohh sure, good catch :)